.idea/modules.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I thought this file was no longer needed. From the comment on #581 but that applies to only projects which do an auto-import. I am not quite sure if it will work on TC.